README.md

```diff
@@ -10,13 +10,18 @@ new beta version available for download.
 
 Installation is just a few clicks. Follow the guide to get started.
 
+## [👉 Building and installing the Safari Web Extension](https://developer.apple.com/documentation/safariservices/safari_app_extensions/building_a_safari_app_extension#2957926)
+
+Follow the linked guide sections to build and unsigned version of the Safari Web Extension and install it in Safari.
```

Comment:
Once we sign and distribute our own we could also add another section below where we direct to each "production release" installation ([chrome app](https://chrome.google.com/webstore/detail/forem/dnncmjdcbcccmddpebibkolaflnakklo) and mac App Store) 🙂

Reply:
I'd like to try to "self-sign" the Forem For Safari app and see if I can get Safari to load the extension that way: that would let us distribute outside the App Store!